mantidproject/mantid
# Sort column maintained across ICat search tabs

Fixes #10167 

To test:
- Open the ICat Search in MantidPlot
- Perform a search that will give many results and so cross onto multiple pages
- Sort results by a column other than StartDate
- Change sort results page
- Confirm the new page of results is sorted by the same column and in the same order (Ascending/Descending)
